grammars/asciidoc.cson

```diff
@@ -30,7 +30,7 @@
   #
   #   === Level 2 Section
   {
-    'begin': '^={1,6}\\s+'
+    'begin': '^={1,6}\\p{Blank}+'
```

Comment:
As you might suspect, I had a trailing tab on a line and realized it can be a common issue, since most text editors don't display tabs except as blank space.